clairmeta/dcp_check_am.py

```diff
@@ -116,18 +117,55 @@ def check_assets_am_path(self, am, asset):
         """ AssetMap assets path validation.
 
             Reference :
-                SMPTE 429-9-2014 7.1
+                SMPTE 429-9-2014 7.1, A.1
         """
-        uuid, path, _ = asset
 
-        if path == '':
-            raise CheckException("Empty path for {}".format(uuid))
+        path_errors = []
+        _, path, _ = asset
+
+        path_segments = list(filter(None, path.split('/')))
```

Comment:
[Minor] Is the `filter` needed here ?

Reply:
Yes. It takes care of eliminating empty path segments in paths like "/dcp//titles/more_titles///film_titles".

Reply:
Good stuff, TIL!